stock_orderpoint_automatic_creation/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Daniel Campos <danielcampos@avanzosc.es> - Avanzosc S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import product
```

Comment:
Are you sure?

In oca module, It's defined product.py and product_template.py:

https://github.com/OCA/OCB/tree/10.0/addons/product/models